cmd/grafana-agent/Dockerfile.windows

```diff
@@ -7,7 +7,7 @@ WORKDIR /src/agent
 
 SHELL ["cmd", "/S", "/C"]
 
-RUN powershell "Invoke-Expression ""C:\'Program Files'\git\bin\bash.exe -c 'RELEASE_BUILD=${RELEASE_BUILD} VERSION=${VERSION} make generate-ui && make agent'"""
+RUN ""C:\Program Files\git\bin\bash.exe"" -c 'RELEASE_BUILD=${RELEASE_BUILD} VERSION=${VERSION} make generate-ui && make agent'
```

Comment:
I noticed this issue when the yarn command timed out on the drone the build continued and failed on the following step when it should have quit right here. TIL when cmd invokes powershell this way it doesn't care how the powershell command does.